src/main/java/com/ipiecoles/java/java220/Employe.java

```diff
@@ -26,6 +28,16 @@ public Employe(String nom, String prenom, String matricule, LocalDate dateEmbauc
         this.salaire = salaire;
     }
 
+    public Employe(String nom, String prenom, String matricule, LocalDate dateEmbauche, Double salaire, Boolean tempsPartiel, String sexe) {
+        this.nom = nom;
+        this.prenom = prenom;
+        this.matricule = matricule;
+        this.dateEmbauche = dateEmbauche;
+        this.salaire = salaire;
```

Comment:
Utilise plutôt le constructeur déjà existant au lieu de dupliquer ces lignes
```suggestion
        this(nom, prenom, matricule, dateEmbauche, salaire);
```